docs/Contributing/API-for-contributors.md

```diff
@@ -1400,4 +1408,308 @@ NOTE: when updating a policy, team and platform will be ignored.
 
 `Status: 200`
 
+### Device-authenticated routes
+
+Device-authenticated routes are routes used by the Fleet Desktop application. Unlike most other routes, they are not authenticated with a Fleet user's API token, they use a device-specific token and as such, are not meant to be consumed by users.
```

Comment:
> are not meant to be consumed by users

They are consumed by Fleet Desktop users (when they click on "My device").

Reply:
Happy to rephrase, but that's not what I mean - when you click on a menu item, you don't deliberately or knowingly consume an API. What I mean is that it's not an endpoint that the users are meant to curl or use in scripts or custom development, unlike the official REST API that they can use with their own API token linked to their Fleet user/account.

Reply:
I'll just remove that final part, it doesn't add much to the documentation anyway.